app/workers/s3_raw.rb

```diff
@@ -23,7 +32,7 @@ def work
       loop do
         messages = queue.receive_messages(max_number_of_messages: 10)
         batched_messages += messages.to_a
-        break if messages.size == 0
+        break if messages.size == 0 || batched_messages.size > 10_000
```

Comment:
Hm... realized this probably needs to be a little smarter. Ideally if we hit a batch limit size we should re-trigger right after the batch is done rather than waiting for the next hour to come around. But data rate is low enough that this should work for now.